annaeliana7/Healthy-Kitchen
# Update README with new information

## What
This pull request updates the README file with new information about the project. The changes include:

- Adding a new section on how to run the project locally
- Updating the installation instructions with new dependencies
- Adding screenshots to the README to better illustrate the project

## Why
The goal of this pull request is to review and implement updates to the project, ensuring that all code adheres to linter standards and guidelines.

## How
Steps to test:

- Clone the repository
- Checkout to this branch
- Follow the instructions to run the project locally
- Verify that the new information is accurate and that the screenshots are helpful

## Note
- Please review the changes made in the README and let me know if there are any other updates that need to be done.
- I will continue to monitor this pull request and make any necessary updates based on feedback.
Thanks!

### Changes details
- Detail one
- Detail two
 
## Missed anything?
- [x] Explained the purpose of this PR.
- [x] Self reviewed the PR.
- [ ] Added or updated test cases.
